src/pvframework/execution.py

```diff
@@ -135,6 +139,53 @@ def info(self) -> _RuntimeExecutionInfo:
         assert self._runtime_execution_info is not None
         return self._runtime_execution_info
 
+    def get_csv_formatted_validator_infos(
+        self,
+        initial_value: str | StringIO | None = None,
+        headings: Optional[Iterable[str]] = (
+            "Manager ID",
+            "Validator function signature",
+            "Mapped fields",
+            "Validator doc string",
+            "Validating type",
+        ),
+    ) -> str:
+        """
+        Returns a csv formatted string containing information about the registered validators.
+        It contains the following information:
+        - Manager ID
+        - Validator function signature
+        - Mapped fields
+        - Validator doc string
+        - Validating type (error, warning)  # TODO: Implement this feature
+
+        You can override the default headings by providing an Iterable of strings. Alternatively, The headings can be
+        omitted by setting `headings` to None.
+        If you specify `initial_value`, the output will be appended to this value. Note, that if you supply a StringIO
+        object, it will modify this object.
+        """
+        if initial_value is None or isinstance(initial_value, str):
+            output = StringIO(initial_value=initial_value)
+        else:
+            output = initial_value
+        csv_writer = csv.writer(output)
+        if headings is not None:
+            csv_writer.writerow(headings)
+        for mapped_validator in self.validators:
+            formatted_doc_string = mapped_validator.validator.func.__doc__
+            if formatted_doc_string:
+                formatted_doc_string = re.sub(r"\n[ \t\r]*", "\\\\n", formatted_doc_string.strip())
+            csv_writer.writerow(
+                (
+                    self.manager_id,
+                    f"{mapped_validator.name}{mapped_validator.validator.signature}",
+                    str(mapped_validator.provision_indicator()),
+                    formatted_doc_string,
+                    "error",
```

Comment:
würde hier eher ein `not implemented yet` statt `error` schreiben

Reply:
Naja, momentan sind alles Errors, das Feature mit den Warnings fehlt halt nur.

Reply:
Um Noah nicht zu verwirren, würde ich es dementsprechend als `error` belassen.